management/project/delete/index.md

```diff
@@ -20,7 +20,7 @@ Once marked, the project enters a **60-day grace period** during which
 - a request for **undeleting** the project can be sent to our [support team](mailto:support@keboola.com).
 
 After the grace period expires, the deletion becomes **irreversible**, and the project along with any associated data
-will be gone for good.
+will be purged and gone for good.

```

Comment:
chtel jsem to jen explicitne zminit, aby az user uvidi v telemetry `auditLog.project.purged` tak aby mu dokumentace nasla toto a doslo mu to. bylo by blbe kdyby to naslo jen tu eventu :)